result.go

```diff
@@ -106,6 +109,14 @@ func (v *ResultErrorFields) Description() string {
 	return v.description
 }
 
+func (v *ResultErrorFields) SetDescriptionFormat(descriptionFormat string) {
+	v.descriptionFormat = descriptionFormat
+}
+
+func (v *ResultErrorFields) DescriptionFormat() string {
```

Comment:
The other fields are explained in the README. Although this is probably less relevant for `DescriptionFormat`, for the sake of completeness we should probably also do it for this one as it will be publicly visible.